src/Jenssegers/Mongodb/Relations/EmbedsMany.php

```diff
@@ -265,25 +265,36 @@ protected function associateExisting($model)
     }
 
     /**
-     * Get a paginator for the "select" statement.
-     * @param int $perPage
-     * @return \Illuminate\Pagination\AbstractPaginator
+     * @param null $perPage
+     * @param array $columns
+     * @param string $pageName
+     * @param null $page
+     * @return \Illuminate\Contracts\Pagination\LengthAwarePaginator
      */
-    public function paginate($perPage = null)
+    public function paginate($perPage = null, $columns = ['*'], $pageName = 'page', $page = null)
```

Comment:
Maybe to add additional test case for this [model]( https://github.com/jenssegers/laravel-mongodb/blob/2d76d22a5ea4dccc6f5a352b0003c1483f72e0f0/tests/models/Address.php#L14)

Reply:
The result will not change.  In the method the only parameter used is `$perPage`. I add the other parameters only to respect the original paginate method.